Author: psykose <alice@ayaya.dev>
Date:   Thu Dec 30 19:05:44 UTC 2021

patch made by psykose <alice@ayaya.dev> to fix build

--- a/src/libs/loguru/meson.build
+++ b/src/libs/loguru/meson.build
@@ -1,6 +1,8 @@
+execinfo_dep = cc.find_library('execinfo', required : true )
+
 libloguru = static_library('loguru',
                            'loguru.cpp',
-                           dependencies : [ threads_dep, dl_dep ])
+                           dependencies : [ threads_dep, dl_dep, execinfo_dep ])
 
 libloguru_dep = declare_dependency(link_with : libloguru,
                                    include_directories : '.')
